force_wfmanager/ui/setup/workflow_tree.py

```diff
@@ -429,6 +429,9 @@ def perform_verify_workflow_event(self):
                 self.workflow_view.mco_view[0]
                 .kpi_view.verify_model_names()
             )
+
+        errors += self.workflow_view.variable_names_registry.verify()
```

Comment:
Obtain errors referring to input and output slots that are raised by generating `Variable` objects